server/models/mtracker.sql

```diff
@@ -0,0 +1,46 @@
+\c mtracker;
+
+DROP TABLE IF EXISTS users CASCADE;
+CREATE TABLE users (
+ userId serial PRIMARY KEY,
+ firstName VARCHAR (50) NOT NULL,
+ lastName VARCHAR (50) NOT NULL,
+ signupDate date NOT NULL DEFAULT CURRENT_DATE,
+ email VARCHAR (50) NOT NULL,
+ pass VARCHAR (20) NOT NULL,
+ isAdmin boolean NOT NULL default false
+);
+
+DROP TABLE IF EXISTS requests;
+CREATE TABLE requests (
+ requestId serial PRIMARY KEY,
+ userId INT NOT NULL,
+ requestDate date NOT NULL DEFAULT CURRENT_DATE,
+ product VARCHAR (20) NOT NULL,
+ requestType VARCHAR (20) NOT NULL,
+ lastCheck date,
+ issue VARCHAR (255) NOT NULL,
+ requestStatus VARCHAR (20) NOT NULL DEFAULT 'pending',
```

Comment:
Is `varchar` the best data type to use here? You have a group of allowed requesStatus, isn't it?